lib/hydra/pcdm/models/concerns/collection_behavior.rb

```diff
@@ -7,6 +7,11 @@ module CollectionBehavior
     included do
       type RDFVocabularies::PCDMTerms.Collection  # TODO switch to using generated vocabulary when ready
       aggregates :members, :predicate => RDFVocabularies::PCDMTerms.hasMember, :class_name => "ActiveFedora::Base"
+
+      def self.collection? collection
```

Comment:
I wonder if this woudn't make more sense in the Hydra::PCDM module. e.g. `Hydra::PCDM.collection?(obj)`